CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## [1.3.0] - T.B.D.
+### Added
+- Cucumber 3 as a dependency
```

Comment:
afaik WebdriverIO doesn't support cucumber v3 yet when running it via wdio-cucumber-framework